requirements.txt

```diff
@@ -29,6 +30,7 @@ pymongo==3.11.3
 pyparsing==2.4.7
 pytest==6.2.2
 python-dateutil==2.8.1
+python-multipart==0.0.5
```

Comment:
Do we still need this after converting from a List of UploadFiles to one UploadFile?

Reply:
we might really soon tbh, not worth taking out